modules/s2/redshift/test/integration/test_VERSION.py

```diff
@@ -0,0 +1,10 @@
+from test_utils import run_query
+
+# Initialize variable just to avoid linter complaints
+__version__ = None
+exec(open('./lib/version.py').read())
```

Comment:
Could you use this here?

```py
from lib._version import __version__
```

Reply:
I don't think so, this still tries to run `__init__.py`.

Took this from this answer: https://stackoverflow.com/a/16084844/8159847

Reply:
True that